examples/samples/messageBrowse.ts

```diff
@@ -0,0 +1,125 @@
+import {
+  OnMessage,
+  OnError,
+  MessagingError,
+  delay,
+  ServiceBusMessage,
+  ReceiveMode,
+  generateUuid,
+  Namespace
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+
+console.log("str: ", str);
+console.log("path: ", path);
+
+async function main(): Promise<void> {
+  // send a set of messages
+  await sendMessage();
+  // browse those messages
+  await peekMessage();
+  // receive the messages [ to clear the queue ]
+  await receiveMessage();
+}
+
+async function sendMessage(): Promise<void> {
+  const nsSend = Namespace.createFromConnectionString(str);
+  const sendClient = nsSend.createQueueClient(path);
+  var data = [
+    { name: "Einstein", firstName: "Albert" },
+    { name: "Heisenberg", firstName: "Werner" },
+    { name: "Curie", firstName: "Marie" },
+    { name: "Hawking", firstName: "Steven" },
+    { name: "Newton", firstName: "Isaac" },
+    { name: "Bohr", firstName: "Niels" },
+    { name: "Faraday", firstName: "Michael" },
+    { name: "Galilei", firstName: "Galileo" },
+    { name: "Kepler", firstName: "Johannes" },
+    { name: "Kopernikus", firstName: "Nikolaus" }
+  ];
+  try {
+    for (let i = 0; i < data.length; i++) {
+      var message = {
+        body: JSON.stringify(data[i]),
```

Comment:
We should use the same method for sending messages like we do in the other samples. I don't think we need to use `JSON.stringify` method here.

Reply:
Will change this to the way we send the messages in other samples.